builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -33,7 +33,7 @@ func TestAccVSphereVirtualMachine_basic(t *testing.T) {
 	if v := os.Getenv("VSPHERE_DATASTORE"); v != "" {
 		datastoreOpt = fmt.Sprintf("        datastore = \"%s\"\n", v)
 	}
-	template := os.Getenv("VSPHERE_TEMPLATE")
+	cloneSource := os.Getenv("VSPHERE_CLONE_SOURCE")
```

Comment:
If `"VSPHERE_CLONE_SOURCE"` is required for the test please check that it is set.  We need to fix that in ALL of our tests 😄